wix-media-v2/wix-media-v2/Files/guides/upload-api.md

```diff
@@ -82,6 +82,8 @@ async function uploadMyFile(uploadUrl, fileContent) {
 }
 ```
 
+## Add Uploaded Media
+In order to add the uploaded media files to your Wix site, use [getFileDescriptor()](https://www.wix.com/velo/reference/wix-media-v2/files/getfiledescriptor) to retrieve the compatable media URI. Pass in `data.file.id` from the [response object](#Response) as the only parameter.  
```

Comment:
What's missing here is the context - in other words, _why_ do I need to call this endpoint? This is a highly unusual situation, and your readers are naturally going to wonder why they're bothering to call another function to retrieve a property that's present in this response already. Hold their hand a little bit, and explain the why, and that this next step is required.